py/vtdb/vtgate_utils.py

```diff
@@ -9,6 +10,25 @@
 BACKOFF_MULTIPLIER = 2
 
 
+def log_exception(exc, keyspace=None, tablet_type=None):
+  """This method logs the exception.
+
+  Args:
+    exc: exception raised by calling code
+    args: additional args for the exception.
+    kwargs: keyword args for the exception.
+
+  """
+  logger_object = vtdb_logger.get_logger()
+
+  shard_name = None
```

Comment:
For vtgatev2 yes, we will deprecate this once vtgate rolls out. I didn't want to pass None in the args since
context is lost then.